### PR TITLE
[e2e] Delete ns and client if not nil to avoid panic

### PR DESCRIPTION
**Pull request by lzhecheng**
#### What type of PR is this?
/kind testing

#### What this PR does / why we need it:
[e2e] Delete ns and client if not nil to avoid panic
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: tests/e2e/network/private_link_service.go [hunk 1]
```diff
@@ -59,27 +59,36 @@ var _ = Describe("Private link service", Label(utils.TestSuiteLabelPrivateLinkSe
...
 	})
 
 	AfterEach(func() {
+		if !strings.EqualFold(os.Getenv(utils.LoadBalancerSkuEnv), string(network.PublicIPAddressSkuNameStandard)) {
+			Skip("private link service only works with standard load balancer")
+		}
+
+		if tc.IPFamily != utils.IPv4 {
+			tc = nil
+			return
```
Comment (jwtty) [inline tests/e2e/network/private_link_service.go:89]: Why return instead of skip here?

Comment (lzhecheng) [reply tests/e2e/network/private_link_service.go]: `Skip` is better. Updated.

**Pull request merged**